packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -517,6 +518,24 @@ function* doUnblockUser(action: ReturnType<typeof unblockUser>) {
   }
 }
 
+function* doReportUser(action: ReturnType<typeof reportUser>) {
```

Comment:
Any reason this can't just call `doBlockUser` under the hood, and then we add some additional analytics here?

Reply:
Good call